_data/simple-icons.json

```diff
@@ -75,6 +75,11 @@
             "hex": "FF5A5F",
             "source": "https://www.airbnb.com"
         },
+        {
+            "title": "Go Language",
+            "hex": "6AD7E5",
+            "source": "https://github.com/golang-samples/gopher-vector"
```

Comment:
Is `golang-samples` official? It seems that https://golang.org/doc/gopher/ is the official source of gopher images (unfortunately only the favicon is available as vector).

If the vectorized version from `golang-samples` accurately matches [the official PNG](https://golang.org/doc/gopher/frontpage.png) then there obviously isn't much point in us re-vectorizing it; but I'd still prefer the official source to be listed (with `golang-samples` mentioned in a commit comment)

Reply:
> Is golang-samples official? It seems that https://golang.org/doc/gopher/ is the official source of gopher images (unfortunately only the favicon is available as vector).

And personally I don't think the favicon is the correct icon to use...

> If the vectorized version from golang-samples accurately matches the official PNG then there obviously isn't much point in us re-vectorizing it; but I'd still prefer the official source to be listed (with golang-samples mentioned in a commit comment)

I feel like the SVG from `golang-samples` is an accurate match, but you should decide for yourself. Either way I will update the source link :+1:

Reply:
I definitely agree that the gopher you've chosen is preferable to the favicon.